content/renderer/input/render_widget_input_handler.cc

```diff
@@ -265,6 +269,21 @@ void RenderWidgetInputHandler::HandleInputEvent(
       processed = widget_->GetWebWidget()->HandleInputEvent(coalesced_event);
   }
 
+#if defined(CASTANETS)
```

Comment:
This changes will make duplicated behavior with line 233 for android codes whenever calling WebInputEvent::IsKeyboardEventType().

Is it OK?

Reply:
Yes you are right. It may cause some conflict. The original condition specifically handles DPAD_CENTER key and text input. Not sure about its background.
I just added a TODO comment regarding the specific handling of DPAD_CENTER key.